routes/contract.js

```diff
@@ -3,22 +3,46 @@ import {
   createContractList,
   findContractByFirstParty,
   createContractDetail,
+  updateContractList,
+  getContractContentByContractNo,
+  updateSecondPartySignature,
+  updateSecondParty,
+  findInProgressContracts,
 } from "../services/contract.js";
 
 const router = express.Router();
 
 /** /contract */
 router.get("/", async (req, res) => {
   try {
-    let contractList = await findContractByFirstParty("AMD");
-    return res.status(201).json({ status: 'success', message: 'Contract created successfully', contractList });
+    let firstParty = req.user.firstName + req.user.lastName;
+    let contractList = await findContractByFirstParty(firstParty);
+    return res.status(201).json({ status: 'success', message: 'Get contract list successfully', contractList });
   } catch (err) {
     console.error(`Error while getting contract list`);
     console.error(err);
     return res.status(500).json({ status : 'error', message : err.message });
   }
 });
 
+/** /contract/countInProgress */
+router.get("/countInProgress", async (req, res) => {
+  try {
+    let firstParty = req.user.firstName + req.user.lastName;
+    let contractList = await findInProgressContracts(firstParty);
+    let numberOfContracts = contractList.length;
+    return res.status(201).json({
+      status: "success",
+      message: "Get contract list successfully",
```

Comment:
You can ignore the message field here, since we have the status we can simply go ahead and look for the data